idseq_dag/engine/pipeline_step.py

```diff
@@ -150,6 +201,8 @@ def thread_run(self):
         with log.log_context("dag_step", v):
             with log.log_context("substep_wait_for_input_files", v):
                 self.wait_for_input_files()
+            with log.log_context("validate_input_files", v):
```

Comment:
should this stick to the same pattern as the other lines, i.e. `log.log_context("substep_validate_input_files", v)`?

Reply:
Yes, thanks.